.github/actions/setup-woocommerce-monorepo/action.yml

```diff
@@ -51,7 +51,9 @@ runs:
 
     - name: Install Node and PHP Dependencies
       shell: bash
-      run: pnpm install ${{ steps.parse-input.outputs.INSTALL_FILTERS }}
+      run: |
```

Comment:
If you're gonna do a filtered install you'll still *always* need turbo installed.